tabpfn_client/prompt_agent.py

```diff
@@ -74,22 +84,23 @@ def prompt_terms_and_cond(cls) -> bool:
             "By using TabPFN, you agree to the following terms and conditions:",
             "Do you agree to the above terms and conditions? (y/n): ",
         ])
+        choice = cls._choice_with_retries(t_and_c, ["y", "n"])
+        return choice.lower() == "y"
 
-        choice = input(cls.indent(t_and_c))
-
-        # retry for 3 attempts until valid choice is made
-        is_valid_choice = False
-        for _ in range(3):
-            if choice.lower() not in ["y", "n"]:
-                choice = input(cls.indent("Invalid choice, please enter 'y' or 'n': "))
-            else:
-                is_valid_choice = True
-                break
+    @classmethod
+    def prompt_add_user_information(cls, user_auth_handler: "UserAuthenticationClient"):
+        print(cls.indent("To help us tailor our support and services to your needs, we have a few optional questions. "
+                         "Feel free to skip any question by leaving it blank.") + "\n")
+        company = input(cls.indent("Where do you work?: "))
```

Comment:
leave the colons (:) away, as you already have a question mark, in all three lines